manpages/pihole.8

```diff
@@ -66,14 +66,24 @@ Available commands and options:
     Adds or removes specified domain or domains to the blacklist
 .br
 
+\fB--regex, regex\fR [options] [<regex1> <regex2 ...>]
+.br
+    Add or removes specified regex filter to the regex blacklist
+.br
+
+\fB--whiteregex\fR [options] [<regex1> <regex2 ...>]
```

Comment:
This should have a hyphen between `white` and `regex`. Same below with `white` and `wild`.